### ISSUE TITLE
result object can't be plotted

### ISSUE THREAD

**Issue by emmyneutron** [abodein/timeOmics_BioC#3]
I am trying to use timeomics package, but I can't get results. I have unevenly spaced time series omics data(gut 16s rRNA , RNA-seq), but samples collected with different time spans. so, I found it so hard to do multi-Omics longitudinal clustering. I got the result object, but I can't plot it as the attached image. could you guide me debugging this issue?

![Screenshot from 2020-09-01 10-22-32](https://user-images.githubusercontent.com/13057587/92322666-ea68f700-f032-11ea-984a-c4759954e17b.png)

**Comment by abodein** [comment]
Hi @emmyneutron 
If nothing appears in your plot, it is probably because the "sample" names are not integer/float.
After the modelling, there is a data reduction and the rownames become 0,1,2, ..., t depending on your times instead of 'sample_1, sample_2, ..."
Try to change the rownames before the clustering or use the option "time" of the plotLong() function.
Don't hesitate if anything goes wrong.

**Comment by odrangel7** [comment]
I'm sorry, but I faced the same issue. So I reopen here- 
First, I see that when modelling and filtering my dataset i obtained only three rows (in my case i only have three timepoints 0,15 and 39). So the filtered dataset contain only three rows. I compared with the example and there the filtered contain A_1, B_1, C_1 and so on (so it filters the columns and keeps all the rows). So I did filter my dataset manually, based on the molecules to keep and jumped to this part. Everything works fine, but when plotting my plots are empty and when using the "time" option, I get the following error: Error in `.rowNamesDF<-`(x, value = value) : 
  duplicate 'row.names' are not allowed
In addition: Warning message:
non-unique values when setting 'row.names': ‘0’, ‘15’, ‘39’ 

Any suggestion? Is my filtering correct when extracting the dataset for the PLS? Which names does the rows should have then ?
Thanks in advance.
